### PR TITLE
Added cards block type.

### PR DESCRIPTION
**Pull request by arshadkhan53**
Removed space.

### CODE REVIEW

File: config/install/fvm.settings.yml [hunk 1]
```diff
@@ -34,3 +35,25 @@ paragraph__embed__view_modes:
...
+  preview: 0
+paragraph__gallery: 0
+paragraph__gallery__view_modes:
+  preview: 0
+paragraph__gallery_carousel: 0
+paragraph__gallery_carousel__view_modes:
+  preview: 0
+paragraph__view: 0
+paragraph__view__view_modes:
+  preview: 0
```
Comment (vaibhavjain-in) [inline config/install/fvm.settings.yml]: I believe, this is incorrect. there are entities in the config which are not installed on default profile install, like smart article. Can you please see.

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@arshadkhan53 Please cater to feedback.

File: config/install/fvm.settings.yml (hunk 1 above)
Comment (arshadkhan53) [reply config/install/fvm.settings.yml]: Addressed

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**